packages/components/src/dropdown-menu/style.scss

```diff
@@ -78,16 +78,24 @@
 			height: 1px;
 		}
 
-		&:hover:not(:disabled):not([aria-disabled="true"]):not(.is-default) {
-			@include formatting-button-style__hover();
+		// Plain menu styles.
+		&:focus:not(:disabled):not([aria-disabled="true"]):not(.is-default) {
+			@include menu-style__focus();
 		}
 
-		&:not(:disabled):not([aria-disabled="true"]):not(.is-default).is-active {
-			@include formatting-button-style__active();
+		// Formatting buttons
```

Comment:
It's not only for formatting buttons though it's for any Dropdown menu containing icons right? (like the table block)

Reply:
Right, but the table block will never have an is-active applied, so the dark background won't be applied.